internal/service/handlers/edit_referrals.go

```diff
@@ -37,24 +39,16 @@ func EditReferrals(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
-	var index uint64
-	if balance != nil {
-		index, err = ReferralsQ(r).FilterByUserDID(balance.DID).Count()
-		if err != nil {
-			Log(r).WithError(err).Error("Failed to get referral count for user DID")
-			ape.RenderErr(w, problems.InternalError())
-			return
-		}
-	}
-
-	referrals, err := adjustReferralsCount(index, req, r)
+	added, err := adjustReferralsCount(req, r)
 	if err != nil {
 		Log(r).WithError(err).Error("Failed to adjust referrals count")
 		ape.RenderErr(w, problems.InternalError())
 		return
 	}
 
-	ape.Render(w, newBalanceResponse(*balance, referrals))
+	ape.Render(w, struct {
```

Comment:
add response struct to the open-api definitions (consider doing that for all responses)

Reply:
This is a private endpoint, so we haven't added it yet